src/service/video-manager-impl.js

```diff
@@ -861,24 +861,44 @@ class VideoEntry {
     });
 
     // Listen to pause, play and user interaction events.
-    const unlistenInteraction = listen(mask, 'click', onInteraction.bind(this));
+    const unlisteners = [];
```

Comment:
they need to unlisten at different times. we remove any autoplay listeners as soon as user interacts.